src/readers/morphologySWC.cpp

```diff
@@ -206,6 +206,14 @@ class SWCBuilder
         const Sample& child1 = _children[0];
         const Sample& child2 = _children[1];
 
```

Comment:
I think you should explain this and put a link to the issue

Reply:
ok